### ISSUE TITLE
Cant stack minecarts in one spot anymore

### ISSUE THREAD

**Issue by SnowyBoi64** [Wilyicaro/Legacy-Minecraft#285]
![image](https://github.com/Wilyicaro/Legacy-Minecraft/assets/80140847/ba43fdbe-5ca6-4b32-95cb-b603812b692d)

![image](https://github.com/Wilyicaro/Legacy-Minecraft/assets/80140847/61fc05e7-7d0f-47fa-990a-7a0c11060185)

i assume the changes to speed up minecarts may have caused this?

**Comment by Wilyicaro** [comment]
And could you do this in LE?

**Comment by SnowyBoi64** [comment]
I believe you could, as it was a method to launch yourself through the world border with the sheer force of the tnt minecarts at one point. But I’ll try on my Xbox one copy to see if it was changed

**Comment by SnowyBoi64** [comment]
Yes, though you can’t hold it down and have to click to stack each one.

**Comment by Wilyicaro** [comment]
Fixed in 1.7.5, thanks for reporting.
